app/controllers/api/v1/apps_controller.rb

```diff
@@ -24,6 +24,6 @@ def app_scopes_or_default
   end
 
   def app_params
-    params.permit(:client_name, :redirect_uris, :scopes, :website)
+    params.permit(:client_name, :scopes, :website, :redirect_uris, redirect_uris: [])
```

Comment:
This actually took me surprisingly long to figure out that I could write it like this.

Reply:
I think we may actually want this to be with `require` instead for several of these. e.g., 
```suggestion
    params.require(:client_name, :redirect_uris, redirect_uris: []).permit(:scopes, :website)
```

i.e., you definitely can't create an application without a `client_name` and `redirect_uris` value.

But `scopes` and `website` are actually optional.

Reply:
Confusingly, this is not how `require` works, see https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require

Reply:
Hmm, okay, well, we may want to think about some better input validation here, since there is potential for this to be an attack surface.

Reply:
I'm not sure; just feels like we should be doing more validation here, rather than relying on the Model validations to catch issues.